packages/@okta/vuepress-site/docs/guides/create-token-with-groups-claim/request-token-claim/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Request a token that contains a custom claim
+---
+The sections in this guide include information on building a URL to request a claim. These sections refer you to this page for the specific steps to build that URL to request a claim and decode the JWT to verify that the claim was included in the token. Specific request and payload examples remain in the appropriate sections. Move on to the section for the claim that you want to create if you don't currently need these steps.
+
+To test the full authentication flow that returns an ID token or an access token, build your request URL:
+
+1. Obtain the following values from your OpenID Connect (OIDC) application, both of which can be found on the application's **General** tab:
+
+    * Client ID
+    * Redirect URI
+
+2. Use the authorization server's authorization endpoint:
+
+    * An Okta Org Authorization Server authorization endpoint looks like this:
+
+        `https://${yourOktaDomain}/oauth2/v1/authorize`
+
+    * A custom authorization endpoint looks like this:
+
+        `https://${yourOktaDomain}/oauth2/${authServerId}/v1/authorize`
+
+    > **Note:** If you add the claim to the default authorization server, the `${authServerId}` is `default`.
+
+    You can retrieve a custom authorization server's authorization endpoint using the server's metadata URI:
+
+    **ID token**
+    `https://${yourOktaDomain}/oauth2/${authServerId}/.well-known/openid-configuration`
+
+    **Access token**
+    `https://${yourOktaDomain}/oauth2/${authServerId}/.well-known/oauth-authorization-server`
+
+3. Add the following query parameters to the URL:
+
+    * Your OIDC application's `client_id` and `redirect_uri`
+    * A scope, which for the purposes of the examples is `openid`. When you are adding a groups claims, both the `openid` and the `groups` scopes are included.
+    * The response type, which for an ID token is `id_token` and an access token is `token`
```

Comment:
The example uses Implict flow. For the Authorization Code flow, the response type should be code. You can then retrieve the ID Token and Access Tokens by exchanging the Authorization code against the token endpoint. (Add required API doc links)